py3status/modules/ping_host.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Display whether a hostname or IP address responds to pings.
+
+This module pings a specified hostname or IP address and displays whether the
+host is "up" or "down". The ping timeout, up text, and down text are configurable.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 2)
+    down_text: text to display if host does not respond (default '{host} is down')
+    host: hostname or IP address to ping (default "www.google.com")
+    ping_timeout: seconds to wait for ping responses (default 2)
+    up_text: text to display if host responds (default '{host} is up')
+
+Format placeholders:
+    {host} The host or IP address set in the "host" configuration parameter.
+
+Examples:
+
+ping_host {
+  host = "www.google.com"
+  ping_timeout = 1
+  up_text "Google is up"
+  down_text "Google is down"
+}
+
+ping_host {
+  host = "myraspberrypi"
+  ping_tieout = 2
+}
+
+SAMPLE OUTPUT
+up
+{
+    'color': '#00FF00',
+    'full_text': u'www.google.com is up',
+    'cached_until': 1510792199.0
+}
+
+down
+{
+    'color': '#FF0000',
+    'full_text': u'www.google.com is down',
+    'cached_until': 1510792199.0
+}
+
+@author Micah Strube <micah.strube@me.com>
+
+"""
+
+from os import system as system_call
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 2
+    down_text = '{host} is down'
+    host = 'www.google.com'
+    ping_timeout = 2
+    up_text = '{host} is up'
+
+    def ping_host(self):
+        ping_cmd = "ping -c 1 -W {t} {h} > /dev/null".format(
+                t=self.ping_timeout,
+                h=self.host)
+
+        is_up = system_call(ping_cmd) == 0
```

Comment:
We can use one of `py3status`'s several helpers. I think `self.py3.command_run` replaces this one.